ru/lessons/basics/enum.md

```diff
@@ -1,25 +1,46 @@
 ---
-version: 0.9.0
+version: 1.2.0
 layout: page
 title: Enum
 category: basics
 order: 3
 lang: ru
 ---
 
-Набор алгоритмов для операций с коллекциями.
+Набор алгоритмов для операций с перечислениями.
```

Comment:
@ruslanshakirov Actually, `перечисление` sounds a little weird for me just because it is a verbal noun meaning the process of enumeration. In my opinion, `перечисление` may perfectly fit when we're talking about a single instance of smth. enumerable.

Discussion with colleagues was pretty controversial and we weren't able to find the best option. Then I tried to find some other translations of related topics and finally decided that the current state seems fine. So let's leave it like it is right now. Sorry for bothering.